lib/server.js

```diff
@@ -302,6 +302,11 @@ class Server extends KarmaEventEmitter {
         }
         executor.schedule()
       })
+
+      this.on('stop', function (done) {
```

Comment:
So 'stop' handler is only defined if config.autoWatch?

Reply:
I thought that made sense, since otherwise the server should stop anyway. Do you think it should always be available?

That would allow stopping a server that's taking too long in a single run. I don't have that usecase myself, but maybe others do.

Reply:
I think having the API go away when the config changes could be unnecessarily mysterious.

Reply:
Yes, you're right. That would be confusing. Moved outside this condition.